### PR TITLE
#128 inline template notes

### LINKED ISSUE #128: Using "/" notation to add notes to a new task with a template doesn't work.
I found out that creating a task with a template and notes doesn't add the notes passed in from the command line.

```bash
dstask add "Test Task" template:123 / This should be a note
```

I could see this as acceptable if the template has default note, but it's for all templates.

### PR DESCRIPTION
**Pull request by cgardner**
Closes #128

### CODE REVIEW

File: task.go [hunk 1]
```diff
@@ -260,6 +260,13 @@ func (t *Task) Modify(query Query) {
 	if query.Priority != "" {
 		t.Priority = query.Priority
 	}
+
+	if query.Note != "" {
```
Comment (naggie) [inline task.go:264]: I think the if statements can be combined, as if query.Note is `""` when appending it to `t.Notes` will have no effect so it doesn't have to be conditional; the only thing that needs to be is the addition of `"\n"`.

Comment (naggie) [reply task.go:264]: Other than that looks good to go. Well tested too. Thanks!

**Comment by dontlaugh** [comment]
@cgardner Anything else on this?

**Comment by cgardner** [comment]
I don't think so

**Comment by naggie** [comment]
Apparently I forgot about this, sorry!

**Pull request merged**